integreat_cms/cms/migrations/0115_update_contact_permission.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from django.db import migrations
+
+from ..constants import roles
+
+if TYPE_CHECKING:
+    from django.apps.registry import Apps
+    from django.db.backends.base.schema import BaseDatabaseSchemaEditor
+
+
+def update_roles(
+    apps: Apps,
+    _schema_editor: BaseDatabaseSchemaEditor,
+) -> None:
+    """
+    Update the permission settings for the model contact
+
+    :param apps: The configuration of installed applications
+    """
+    # We can't import the Person model directly as it may be a newer
+    # version than this migration expects. We use the historical version.
+    Group = apps.get_model("auth", "Group")
+    Permission = apps.get_model("auth", "Permission")
+
+    # Assign the correct permissions
+    for role_name in dict(roles.CHOICES):
+        group, _ = Group.objects.get_or_create(name=role_name)
+        # Clear permissions
+        group.permissions.clear()
```

Comment:
Why do we need this step? :)

Reply:
@JoeyStk I actually copy&pasted from #1988 :see_no_evil: but so far I understood, it's safer to once clear and re-assign the permission of all roles instead of changing/modifying partially, as this always works regardless of how the role and permission are set at the moment this migration is run.